lib/transform-directives-for-browser.js

```diff
@@ -9,36 +36,33 @@ var handlers = {
       var basePolicy = {}
       if (version < 5) {
         basePolicy.allow = ['*']
+
+        if (directives.defaultSrc) {
+          basePolicy.allow = directives.defaultSrc
+          delete directives.defaultSrc
+        }
       } else {
         basePolicy.defaultSrc = ['*']
       }
 
-      return reduce(directives, function (result, value, key) {
-        if (key === 'connectSrc') {
-          result.xhrSrc = value
-        } else if ((key === 'defaultSrc') && (version < 5)) {
-          result.allow = value
-        } else {
-          result[key] = value
-        }
-
-        if (key === 'scriptSrc') {
-          var optionsValues = []
-
-          if (value.indexOf("'unsafe-inline'") !== -1) {
-            optionsValues.push('inline-script')
-          }
-          if (value.indexOf("'unsafe-eval'") !== -1) {
-            optionsValues.push('eval-script')
-          }
+      return createFirefoxPreCSP10Directives(directives, basePolicy, version)
```

Comment:
It looks like that last argument isn't used by `createFirefoxPreCSP10Directives`. Is that right?